test/specs/components/Icon/Icon-test.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react'
+import { isConformant, handlesAccessibility, getRenderedAttribute } from '../../commonTests'
+
+import Icon from '../../../../src/components/Icon/Icon'
+import { MenuBehavior } from 'src/lib/accessibility'
+import { getTestingRenderedComponent } from 'test/utils'
+
+describe('Icon', () => {
+  isConformant(Icon)
+
+  describe('Icon accessibility', () => {
+    handlesAccessibility(Icon, {
+      defaultRootRole: undefined,
+      accessibilityOverride: MenuBehavior,
+      overriddenRootRole: 'menu',
+    })
+
+    describe('aria hidden', () => {
```

Comment:
it is better to introduce test for SVG icon as well, as the problem with unassigned `aria-hidden` attribute has not been caught by tests now